src/main/java/com/jfrog/ide/idea/ui/LocalComponentsTree.java

```diff
@@ -163,4 +170,25 @@ public void actionPerformed(ActionEvent e) {
             }
         });
     }
+
+    public void cacheTree() throws IOException {
+        SortableChildrenTreeNode root = (SortableChildrenTreeNode) getModel().getRoot();
+        cache.cacheNodes(Collections.list(root.children()).stream().map(treeNode -> (FileTreeNode) treeNode).collect(Collectors.toList()));
```

Comment:
More efficient - removes 2 passes on the root children:
```suggestion
        //noinspection unchecked
        cache.cacheNodes((List<FileTreeNode>) root.children());
```

Reply:
It doesn't work because root.children() is Enumeration, so I found another solution:
```java
//noinspection unchecked
cache.cacheNodes((List<FileTreeNode>)(List<?>) Collections.list(root.children()));
```